packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -72,6 +72,7 @@ export interface IVerticalBarChartState extends IBasestate {
   hoverXValue?: string | number | null;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33510&commitId=ed027b225ae82f5213bcdb883fb8e287324d4361&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|
|react-charting-LineChart.Events.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>